CONTRIBUTING.md

```diff
@@ -1,127 +1,30 @@
-# Contributing to Ground Platform
+# Contributing to Ground
 
-This document covers everything you need to know about contributing to Ground
-Platform. Before you get started, please make sure you read all of the 
-sections and are able to sign our CLA.
+This document provides details on how to get started contributing to the
+project. Please read this document carefully and be sure to sign the [Google
+ Contributor License Agreement (CLA)](https://cla.developers.google.com)
+ before you begin.
 
-## Contributor License Agreement (CLA)
-
-Contributions to this project must be accompanied by a Contributor License
-Agreement. You (or your employer) retain the copyright to your contribution;
-this simply gives us permission to use and redistribute your contributions as
-part of the project. Head over to <https://cla.developers.google.com/> to see
-your current agreements on file or to sign a new one.
-
-You generally only need to submit a CLA once, so if you've already submitted one
-(even if it was for a different project), you probably don't need to do it
-again.
+This project follows
+[Google's Open Source Community Guidelines](https://opensource.google.com/conduct/).
+Take some time to familiarize yourself with these guidelines before
+contributing.
 
-## Code reviews
+## Submitting code
 
 Before starting work on a change, comment on one of the [open issues](https://github.com/google/ground-platform/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen)
 saying you'd like to take it on. If one does not exist, you can also
 [create one here](https://github.com/google/ground-platform/issues/new).
 
-Be sure to reference this issue number in your pull request (e.g., 
-"Fix truncated text values #27"), and preferably in your commit message.
+Once an issue has been assigned to you, you may start writing your own code.
+The first time you contribute, be sure to must [create a fork](https://github.com/google/ground-platform/fork)
+of the `ground-platform` repostory. You may then
+[clone your new fork](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+to your local workspace and begin editing.
 
-All submissions, including submissions by project members, require review. We
+All submissions require review and approval by at least one maintainer. We
```

Comment:
We might also want to mention "requesting review" on the PR here so that maintainers are notified. We should also make sure anyone can do so -- I believe a repo setting controls who can request reviews.